pkg/kubelet/controller.go

```diff
@@ -278,9 +279,11 @@ func (na *nodeAddress) discoveryV1Endpoint() discoveryv1.Endpoint {
 		Conditions: discoveryv1.EndpointConditions{
 			Ready: ptr.To(true),
 		},
+		NodeName: ptr.To(na.name),
 		TargetRef: &v1.ObjectReference{
 			Kind:       "Node",
 			Name:       na.name,
+			Namespace:  na.namespace,
```

Comment:
Node resource is not namespace-scoped and it should always be empty.

Reply:
Thanks. Got it. 
Remove it.